docs/customization.md

```diff
@@ -507,6 +507,7 @@ Each `Change` has the following fields:
 | scope | `str | None` | An optional scope |
 | message | `str` | The commit message body |
 | sha1 | `str` | The commit `sha1` |
+| parents | `list[str]` | The parent commit(s) `sha1`(s) |
```

Comment:
Please add more information to this section, specially what's it useful for:

> Example usage: generate changelogs based on merge commits only, or listing commits that belong to the same PR under its own section.

Reply:
Added similar phrasing just below this table after the note. Let me know if that's ok.